Integrations/Securonix/Securonix.py

```diff
@@ -888,6 +1023,76 @@ def get_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
     return human_readable, entry_context, watchlist
 
 
+def create_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Create a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    watchlist_name = args.get('watchlist_name')
+
+    response = client.create_watchlist_request(watchlist_name)
+
+    if 'successfully' not in response:
+        raise Exception(f'Failed to list watchlists.')
+    human_readable = f'Watchlist {watchlist_name} was created successfully.'
+    entry_context = {f'Securonix.Watchlists(val.Watchlistname === obj.Watchlistname)': watchlist_name}
+    return human_readable, entry_context, response
+
+
+def check_entity_in_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Check if entity is in a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    entity_name = args.get('entity_name')
+    watchlist_name = args.get('watchlist_name')
+    watchlist = client.check_entity_in_watchlist_request(entity_name, watchlist_name)
+
+    result = watchlist.get('result')
+    if result == 'NO' or (isinstance(result, list) and result[0] == 'NO'):
+        human_readable = f'Entity unique identifier {entity_name} provided is not on the watchlist: {watchlist_name}.'
+        output = {'Entityname': entity_name}
+    else:  # YES
+        human_readable = f'The Entity unique identifier {entity_name} provided is on the watchlist: {watchlist_name}.'
+        output = {
+            'Entityname': entity_name,
+            'Watchlistname': watchlist_name
+        }
+    entry_context = {f'Securonix.EntityInWatchlist(val.Entityname === obj.Entityname)': output}
+    return human_readable, entry_context, watchlist
+
+
+def add_entity_to_watchlist(client: Client, args) -> Tuple[str, Dict, Dict]:
+    """Adds an entity to a watchlist.
+
+    Args:
+        client: Client object with request.
+        args: Usually demisto.args()
+    Returns:
+        Outputs.
+    """
+    watchlist_name = args.get('watchlist_name')
+    entity_type = args.get('entity_type')
+    entity_name = args.get('entity_name')
+    expiry_days = args.get('expiry_days') if 'expiry_days' in args else '30'
+
+    response = client.add_entity_to_watchlist_request(watchlist_name, entity_type, entity_name, expiry_days)
+
+    if 'successfull' not in response:
+        raise Exception(f'Failed to add entity {entity_name} to the watchlist {watchlist_name}.\n'
```

Comment:
maybe add response in the Exception? or is there a reason? need to see how dose it look.